store/transfer_processor.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/getamis/eth-indexer/client"
+	common2 "github.com/getamis/eth-indexer/common"
```

Comment:
let's use `gethcommon "github.com/ethereum/go-ethereum/common"`, then you can represent `"github.com/getamis/eth-indexer/common"` as common?

Reply:
done